tox.ini

```diff
@@ -35,38 +45,35 @@ commands =
 # Run Python unit tests with all dependencies installed (ase & chemfiles are
 # optional dependencies)
 deps =
-    equistore @ https://github.com/lab-cosmo/equistore/archive/51fc78c.zip
-    numpy
-    ase
-    chemfiles
-
-    discover
-
+    {[testenv]deps}
+    {[testenv]extra_deps}
 commands =
     python setup.py --quiet install
     discover -p "*.py" -s python/tests
 
 
 [testenv:min-deps]
 # Run Python unit tests with the minimal dependencies installed
-deps =
-    equistore @ https://github.com/lab-cosmo/equistore/archive/51fc78c.zip
-    numpy
-
-    discover
-
+deps = {[testenv]deps}
 commands =
     python setup.py --quiet install
     discover -p "*.py" -s python/tests
 
+[testenv:docs]
+allowlist_externals =
+    sphinx-build
```

Comment:
This should not be needed, tox should be using sphinx-build from its own virtualenv